src/consensus/pbft/libbyz/replica.cpp

```diff
@@ -2001,8 +2006,8 @@ void Replica::process_new_view(Seqno min, Digest d, Seqno max, Seqno ms)
     if (primary() == id())
     {
       ByzInfo info;
```

Comment:
why do we set the nonce to `0` when we add the pre-prepare to the `pc` but not when we add the prepare to the `pc`?

Reply:
Is the nonce set correctly here on the `pp` message? Since now the nonce is included in pre-prepare and will go onto the ledger, I am just concerned about getting misaligned merkle roots here.

Reply:
I put the response to the question as a comment in the code

Reply:
I rewrote a bunch and now this comment no longer applies